test/lib/d.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
I'm not super keen on sticking fixtures in the test directory, because they have a tendency to get confusing as to which tests they go with once there are a lot of them. Can you give a go at converting these into something inline in the test, say with [`tacks`](https://github.com/iarna/tacks#classes)– you can see an example of that usage over in npm:

https://github.com/npm/npm/blob/master/test/tap/legacy-array-bin.js

The `tacks` binary included with the library can generate the boilerplate `Dir` & `File` sections from these existing files.